interfaces/apparmor/spec_test.go

```diff
@@ -356,3 +356,42 @@ func (s *specSuite) TestChopTree(c *C) {
 		}
 	}
 }
+
+const snapTrivial = `
+name: some-snap
+version: 0
+apps:
+  app:
+    command: app-command
+`
+
+func (s *specSuite) TestApparmorParallelInstanceSnippetsNotInstanceKeyed(c *C) {
+	snapInfo := snaptest.MockInfo(c, snapTrivial, &snap.SideInfo{Revision: snap.R(42)})
+	restore := apparmor.SetSpecScope(s.spec, []string{"snap.some-snap.app"})
+	defer restore()
+
+	s.spec.AddOvername(snapInfo)
+	c.Assert(s.spec.Snippets(), HasLen, 0)
+	// non instance-keyed snaps require no extra snippets
+	c.Assert(s.spec.UpdateNS(), HasLen, 0)
+}
+
+func (s *specSuite) TestApparmorParallelInstanceSnippets(c *C) {
```

Comment:
Can you please apply `s/Parallelinstance/Overname/` to test function names for consistency with `Overname` as a new method.